### PR TITLE
Add warnings and remove irrelevant information.

### PR DESCRIPTION
**Pull request by nicosabena**
* Added some warnings on using IdP-initiated flows
* Consolidated instructions for IdP-Initiated setup on SAML connections on one page
* Removed irrelevant instructions for configuring Auth0.js on SAML applications.

### CODE REVIEW

File: articles/protocols/saml/idp-initiated-sso.md [hunk 1]
```diff
@@ -43,4 +63,31 @@ When using **IdP-Initiated SSO**, please make sure to include the `connection` p
 
 ## Lock/Auth0.js
 
-<%= include('../../_includes/_enable_idp_initiated.md') %>
+If your application is a Single Page Application that uses Lock or Auth0.js to process the authentication results, you will have to explicitely indicate that you want to allow IdP-Initiated flows (and thus [open the application to possible Login CSRF attacks](#risks-of-using-an-idp-Initiated-SSO-flow)).
```
Comment (gkwang) [inline articles/protocols/saml/idp-initiated-sso.md]: typo: explicitly

File: articles/protocols/saml/saml-configuration/special-configuration-scenarios/idp-initiated-sso.md [hunk 2]
```diff
@@ -39,22 +35,20 @@ For example, an organization might set up a portal to ensure that users navigate
...
+In general, it is recommended to use Service-Provided flows whenever available. When an application actively requests an authentication as a first step (as is the case for SP-Initiated flows), it can check that the authentication response matches the original request. 
+In IdP-initiated scenarios, on the other hand, the application can not verify that the user actually started the flow. Because of this, the IdP-Initiated flow opens the possibility of an [Login CSRF attack](https://support.detectify.com/customer/portal/articles/1969819-login-csrf), where an attacker can trick a legitimate user into using a session created by the attacker. 
 
-If Auth0 acts as the service provider, you need to make the following changes to support IdP-initiated SSO:
+Login CSRF attacks are generally less of a concern in enterprise-only scenarios, as any would-be attacker would have to come from the same directory of users. But it's definitely a security concern in consumer-facing applications, so we strongly advise against enabling IdP-Inititated flows on SAML connections.
```
Comment (gkwang) [inline articles/protocols/saml/saml-configuration/special-configuration-scenarios/idp-initiated-sso.md:43]: Is this block necessary? I'm wondering if this'll give readers a false sense of security.

File: articles/protocols/saml/saml-configuration/special-configuration-scenarios/idp-initiated-sso.md [hunk 3]
```diff
@@ -68,3 +62,5 @@ If Auth0 acts as the identity provider, you need to make the following changes t
   https://${account.namespace}/samlp
   /${account.clientId}?RelayState=http://FINAL_DESTINATION_URL
```
Comment (gkwang) [inline articles/protocols/saml/saml-configuration/special-configuration-scenarios/idp-initiated-sso.md]: Can you change the RelayState to `https://`

Comment (nicosabena) [reply articles/protocols/saml/saml-configuration/special-configuration-scenarios/idp-initiated-sso.md]: done

File: articles/protocols/saml/idp-initiated-sso.md (hunk 1 above)
Comment (nicosabena) [reply articles/protocols/saml/idp-initiated-sso.md]: thanks, fixed

**Review by gkwang — DISMISSED** [review]
looks good from security's side.

File: articles/protocols/saml/saml-configuration/special-configuration-scenarios/idp-initiated-sso.md (hunk 2 above)
Comment (nicosabena) [reply articles/protocols/saml/saml-configuration/special-configuration-scenarios/idp-initiated-sso.md:43]: Good point. Just changed the message to be more clear.

**Review by erjohnson — APPROVED** [review]

**Pull request merged**